torchrl/modules/tensordict_module/rnn.py

```diff
@@ -23,6 +25,284 @@
 from torchrl.objectives.value.utils import _get_num_per_traj_init
 
 
+class LSTMCell(RNNCellBase):
+    r"""A long short-term memory (LSTM) cell that performs the same operation as nn.LSTMCell but is fully coded in Python.
+
+    .. note::
+        This class is implemented without relying on CuDNN, which makes it compatible with :func:`torch.vmap`
+        and :func:`functorch.vmap`.
```

Comment:
functorch.vmap should not be used, and this will likely not be removed when it'll be completely deprecated.
```suggestion
        This class is implemented without relying on CuDNN, which makes it compatible with :func:`torch.vmap` and :func:`torch.compile`.
```